app/Models/Beatmapset.php

```diff
@@ -84,6 +84,7 @@ class Beatmapset extends Model
 
     const NOMINATIONS_PER_DAY = 3;
     const QUALIFICATIONS_PER_DAY = 6;
```

Comment:
8 maps are moved from qualified -> ranked per day

Reply:
Also these variables are named... wrong (my bad), `NOMINATIONS_PER_DAY` should actually be `QUALIFICATIONS_PER_DAY` and `QUALIFICATIONS_PER_DAY` should be `RANKED_PER_DAY`